crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -917,18 +917,21 @@ pub(crate) fn job_progress_bar(
 }
 
 pub fn print_job_output(
-    mut job: JobDetail,
-    task_selector: Option<Selector>,
+    tasks: Vec<JobTaskInfo>,
     output_stream: OutputStream,
     task_header: bool,
+    task_paths: TaskToPathsMap,
 ) -> anyhow::Result<()> {
-    let task_to_paths = resolve_task_paths(&job);
+    if tasks.is_empty() {
```

Comment:
Please move this to `job.rs`, this code should print the output even if the task list is empty.